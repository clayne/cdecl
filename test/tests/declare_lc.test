cdecl | | declare x as long char | 65
