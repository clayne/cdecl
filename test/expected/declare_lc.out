declare x as long char
        ^
9: error: "long char" is illegal for built-in type
